src/main/java/com/github/tomakehurst/wiremock/admin/AdminUriTemplate.java

```diff
@@ -16,86 +16,69 @@
 package com.github.tomakehurst.wiremock.admin;
 
 import com.github.tomakehurst.wiremock.admin.model.PathParams;
+import com.google.common.base.Function;
 import com.google.common.base.Objects;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.String.format;
 
 public class AdminUriTemplate {
-
-    public static final Pattern PATH_VARIABLE_REGEX = Pattern.compile("^\\{(.*)\\}$");
```

Comment:
The removal of this constant may cause breakage - shall we mark it as deprecated, and not use it, instead?

Reply:
Regarding its purpose not sure, it's supposed to be part of public API and reused. As consequence, I moved the new pattern from `public` to `package-protected`